app/models/user.rb

```diff
@@ -25,17 +25,20 @@ def pending_friends
 
   # Users who have requested to be friends
   def friend_requests
-    inverse_friendships.map { |friendship| friendship.user unless friendship.confirmed }.compact
+    # inverse_friendships.map { |friendship| friendship.user unless friendship.confirmed }.compact
+    Friendship.all.where(friend_id: id, confirmed: false)
   end
 
   def confirm_friend(user)
-    friendship = inverse_friendships.find { |my_friendship| my_friendship.user == user }
+    # friendship = inverse_friendships.find { |my_friendship| my_friendship.user == user }
```

Comment:
Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.